e2e/src/test/java/org/hyades/e2e/BomUploadProcessingE2ET.java

```diff
@@ -225,4 +239,51 @@ private void verifyWebhookNotification() {
         );
     }
 
-}
\ No newline at end of file
+    private void verifyProjectVulnAnalysisCompleteNotification() {
+        wireMock.verify(postRequestedFor(urlPathEqualTo("/notification"))
+                .withRequestBody(equalToJson("""
+                        {
+                           "notification" : {
+                             "level" : "LEVEL_INFORMATIONAL",
+                             "scope" : "SCOPE_PORTFOLIO",
+                             "group" : "GROUP_PROJECT_VULN_ANALYSIS_COMPLETE",
+                             "timestamp" : "${json-unit.any-string}",
+                             "title" : "Project vulnrability analysis complete",
+                             "content" : "${json-unit.any-string}",
+                             "subject" : {
+                               "project" : {
+                                 "uuid": "${json-unit.any-string}",
+                                 "name" : "foo",
+                                 "version" : "bar"
+                               },
+                               "componentAnalysisComplete" : [ {
```

Comment:
The naming is a little odd as it doesn't really describe what the array is representing. Perhaps something like `findings` would be more fitting?

Reply:
fixed :)